focal.yaml

```diff
@@ -54,3 +54,8 @@ commandTests:
     command: "dotnet"
     args: ["--version"]
     expectedOutput: ["6.0"]
+
+  - name: "Musl library and its tools are available"
+    command: "musl-gcc"
+    args: ["--version"]
+    exitCode: 0
```

Comment:
can you please assert the version in the output as well? (you can use regexp as well)

Reply:
Yes, it is the gcc version, but still makes sense.